apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/en.ftl

```diff
@@ -0,0 +1,2 @@
+next-checkout-error-boundary-retry-button = Try again
```

Comment:
Since these are new error messages, you can remove the `next-` from the start of the id.

Reason for this is buried away somewhere in documentation, but the short version is that `next-` is just used for messages that we'll be transferring from `fxa-payments-server` at some point in the future.

Reply:
Could we reuse `payment-error-retry-button = Try again` (and therefore we'd use `next-payment-error-retry-button = Try again` here)?

Reply:
Could this be left as a separate string id, but I'd move the file to `apps/payments/next/app/[locale]` so co-located with the error.tsx where it's used?

Reply:
thanks for the catch @StaberindeZA, it got left behind in a move.